Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -231,9 +245,10 @@ namespace O3DE::ProjectManager
         gemDetailsLayout->addWidget(m_licenseURL);
 
         m_userDefinedGemTags = new FormLineEditWidget(tr("User-defined Gem Tags (Comma separated list)"), "");
+        m_userDefinedGemTags->lineEdit()->setValidator(new QRegularExpressionValidator(QRegularExpression("(,|\\w)+"), this));
```

Comment:
do we also need to consider whitespace characters for tags?
For example, I find myself sometimes writing out comma separated fields like:
`tag1, tag2, tag3`

Reply:
it would be nice for users to not be penalized for adding whitespace(s) before/after a comma, but I consider it a minor thing